### DISCUSSION TITLE
Apple Silcon no pid file

### DISCUSSION THREAD

**Discussion by Austinddan** [jhuckaby/Cronicle#573] [General]
Is Cronicle compatible with apple silicon? Ive followed the install instructions but get a "no pid file" error. Haven't had this issue with my other intel macs. 

I can run the start command and it says the cronicle daemon is started, but when i run the status command it says "cronicle daemon is not running (no pid file)".

Just curious if I missed something or if its just not compatible.

**Comment by jhuckaby** [comment]
I am moving this to a discussion thread.

**Comment by jhuckaby** [comment]
It works just fine for me.  Not only do I develop Cronicle on a M1 MacBook, the installer works too:

```
root@joemax cronicle # bin/control.sh setup

Setup completed successfully!
This server (joemax.local) has been added as the single primary master server.
An administrator account has been created with username 'admin' and password 'admin'.
You should now be able to start the service by typing: '/opt/cronicle/bin/control.sh start'
Then, the web interface should be available at: http://joemax.local:3012/
Please allow for up to 60 seconds for the server to become master.

root@joemax cronicle # 
root@joemax cronicle # bin/control.sh start
bin/control.sh start: Starting up Cronicle Daemon...
bin/control.sh start: Cronicle Daemon started
root@joemax cronicle # 
root@joemax cronicle # 
root@joemax cronicle # bin/control.sh status
status: Cronicle Daemon running (pid 66943)
```

Make sure you are running as root, and you installed to `/opt/cronicle`.  The system expects the PID file to live here, exactly: `/opt/cronicle/logs/cronicled.pid`.  It cannot live anywhere else on the filesystem.
